### PR TITLE
Modifying alerting logic based on alert classifications

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bugid: Tendrl/gluster-integration#452
tendrl-bugid: Tendrl/gluster-integration#449
Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

File: tendrl/gluster_integration/sds_sync/__init__.py [hunk 1]
```diff
@@ -510,7 +513,10 @@ def sync_volumes(volumes, index, vol_options):
                         msg,
                         instance,
                         'WARNING' if current_status == 'Stopped'
-                        else 'INFO'
+                        else 'INFO',
+                        tags={"entity_type": RESOURCE_TYPE_BRICK,
+                              "volume_name": volumes['volume%s.' 'name' % index]
```
Comment (nnDarshan) [inline tendrl/gluster_integration/sds_sync/__init__.py]: "volume_name": volumes['volume%s.name' % index]  ?

Comment (GowthamShanmugam) [reply tendrl/gluster_integration/sds_sync/__init__.py]: to get volume name using index., referred this line https://github.com/GowthamShanmugam/gluster-integration/blob/a5f1a74937d7ff03dd3857705e415ee33bdfde86/tendrl/gluster_integration/sds_sync/__init__.py#L359

**Comment by GowthamShanmugam** [comment]
@shtripat @nnDarshan @r0h4n please review

**Comment by GowthamShanmugam** [comment]
@r0h4n more info added in Tendrl/gluster-integration#452

**Pull request merged**